lib/pronto/config_file.rb

```diff
@@ -31,6 +31,7 @@ class ConfigFile
       'runners' => [],
       'formatters' => [],
       'max_warnings' => nil,
+      'warnings_per_review' => nil,
```

Comment:
@dimitrovv maybe we should have a high default? What do you think?

Reply:
yes, it is possible, I was 🤔not to force this for all so someone can continue using the current flow.
Ex, if not supplied, it will be nil and the comments will not be separated into different PRs. However we've noticed Github is taking too much time and no review is posted if many comments are submitted at once, so we could change it to 30 I guess - would be enough, what do you think?